### ISSUE TITLE
no complaints about "from collections import OrderedDict as OrderedDict"

### ISSUE THREAD

**Issue by bmintz** [PyCQA/pylint#2052]
pylint does not complain if you use `import as` but don't change the name.

### Steps to reproduce
Run pylint on this file:
```py
"""frobnicate"""

from collections import OrderedDict as OrderedDict


SOME_DICT = OrderedDict()
```

### Current behavior

10/10 rating

### Expected behavior

Less than 10/10 rating

### pylint --version output

```
No config file found, using default configuration
pylint 1.8.4, 
astroid 1.6.3
Python 3.6.3 (default, Oct  3 2017, 21:45:48) 
[GCC 7.2.0]
```

**Comment by sushobhit27** [comment]
@bmintz I found your issue to be relevant enough to fix and therefore raised a pull request (https://github.com/PyCQA/pylint/pull/2060) for this issue but it failing due to some non related failing test case.
If you want, I can provide it as a plugin also, if the pull request does not get accepted.

**Comment by PCManticore** [comment]
Why would it warn about this? Yes, the aliasing is redundant, but at the same time this code is innocent enough to not cause any errors. I don't think it warrants a core pylint check, but as @sushobhit27 mentioned, this can be a third party plugin as well.
